src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,167 @@
+.header {
+   padding: 10px 10%;
```

Comment:
same as above

Reply:
removed header class from semantic header tag and done similar changes as Footer component